src/Parser/BalanceDruid/Modules/Features/CastEfficiency.js

```diff
@@ -13,27 +12,26 @@ class CastEfficiency extends CoreCastEfficiency {
       spell: SPELLS.NEW_MOON,
       category: CastEfficiency.SPELL_CATEGORIES.ROTATIONAL,
       getCooldown: (haste, combatant) => {
-        const cd = (combatant.owner.fightDuration / 1000) / (((combatant.owner.fightDuration / 1000) - 2) / 45);
-        return cd;
+        const availableCasts = combatant.owner.modules.newmoon.nmAvailableCasts;
+        return ((combatant.owner.fightDuration / 1000) / availableCasts);
```

Comment:
One set of parenthesis can be omitted 😅

Reply:
You are having fun today picking apart my code, hm? xDD

Reply:
just being thorough, approved the changes though so you can merge without fixing if you so desire.